charts/prometheus-pushgateway/templates/_helpers.tpl

```diff
@@ -77,6 +77,10 @@ Returns pod spec
       imagePullSecrets:
 {{ toYaml .Values.imagePullSecrets | indent 8 }}
     {{- end }}
+      {{- if .Values.extraInitContainers }}
+      initContainers:
+{{ toYaml .Values.extraInitContainers | indent 8 }}      
```

Comment:
```suggestion
        {{ toYaml .Values.extraInitContainers | nindent 8 }}      
```

Reply:
No where else in this file seems to be using nindent does it make sense to create a different pattern?

Reply:
"nindent" seems to be the default nowdays used by "helm create" command.
Therefore i would stick to that default.

Reply:
Ok makes sense. Done